install_config/cluster_metrics.adoc

```diff
@@ -720,6 +720,32 @@ Information about scaling cluster metrics capabilities is available in the
 xref:../scaling_performance/scaling_cluster_metrics.adoc#cluster-metrics-scaling-openshift-metrics-pods[Scaling and
 Performance Guide].
 
+[[metrics-logging]]
+== Integration with Aggregated Logging
+
+Hawkular Alerts might be connected to the Aggregated Logging's Elasticsearch to react on log events. By default,
+Hawkular will try to find Elasticsearch on its default place (namespace `logging`, pod `logging-es`) at every boot.
+If the Aggregated Logging is installed after Hawkular, the Hawkular Metrics pod might need to be restarted in order
+to recognize the new Elasticsearch server. The Hawkular boot log provides a clear indication if the integration could
+not be properly configured, with messages like:
+
+> Failed to import the logging certificate into the store. Continuing, but the logging integration might fail.
```

Comment:
We can place these error messages in code blocks using ----. For example:

     ----
    > Failed to import the logging certificate into the store. Continuing, but the logging integration might fail.
     ----